pom.xml

```diff
@@ -232,7 +232,9 @@
 	</dependencies>
 
 	<properties>
-		<skipTests>true</skipTests>
+	  <skipTests>false</skipTests>
```

Comment:
If the test isn't testing the retrieval process then the file content should be already on disk.